pkg/manager/manifests/templates/pre-delete-job-hypershift.yaml

```diff
@@ -21,8 +21,6 @@ spec:
       - name: {{ .AddonName }}
         image: {{ .Image }}
         imagePullPolicy: {{ .Values.global.imagePullPolicy }}
```

Comment:
please check the `.Values.global.imagePullPolicy` and `.Values.addonInstallNamespace`, I am not sure we can get it.